src/test/rpc/LedgerData_test.cpp

```diff
@@ -221,15 +222,64 @@ class LedgerData_test : public beast::unit_test::suite
         BEAST_EXPECT( running_total == total_count );
     }
 
+    void testLedgerHeader()
+    {
+        using namespace test::jtx;
+        Env env { *this };
+        env.fund(XRP(100000), "alice");
+        env.close();
+
+        // Ledger header should be present in the first query
+        {
+            // Closed ledger with non binary form
+            Json::Value jvParams;
+            jvParams[jss::ledger_index] = "closed";
+            auto jrr = env.rpc("json", "ledger_data",
+                jvParams.toStyledString())[jss::result];
```

Comment:
this `toStyledString()` thing was bad form when I wrote some of these tests...I should have just used `to_string(jvParams)`...feel free to fix that if you are inclined.